src/Pages/Events/event-page.css

```diff
@@ -1,55 +1,90 @@
-.event-desc {
-  margin-top: 10px;
-}
-
 .event-list {
-  padding-top: 10px;
-  padding-bottom: 10px;
+  padding-bottom: 7vh;
+  min-width: max-content;
 }
 
 .event-background {
-  background-color: #ecebeb;
+  background-image: linear-gradient(to right, rgb(4, 27, 56) , rgb(33, 139, 172));
   min-height: 100vh !important;
 }
 
-.event-container {
-  background-color: #ffffff;
-  border-radius: 5px;
-  margin: 10px;
-  transition: ease 0.2s;
-}
-
-.event-container:hover {
-  background-color: #cfcfcf;
+.event-card-button {
+  width: 80%;
+  width: 60vw;
+  height: 150px;
+  height: 20vh;
+  margin-top: 2.5%;
+  border-radius: 10px;
+  background: #e2e6eb;
+  margin-left: 0px;
 }
 
-.card-row {
-  margin: 0px !important;
+.event-title {
+  font-style: normal;
+  font-weight: bold;
+  font-size: 2vw;
+  padding-left: 5.9vw;
+  color: #000000;
+  justify-content: center;
```

Comment:
If you strongly believe in centering, may I suggest making the title's font larger?